include/drivers/st25r3911b_nfca.h

```diff
@@ -157,6 +157,10 @@ struct st25r3911b_nfca_cb {
 	 *  @param tag_sleep If set, the tag is in sleep mode.
 	 */
 	void (*rx_timeout)(bool tag_sleep);
+
+	/** @brief NFC-A tag sleep success.
```

Comment:
Success? Can a failure occur as well in this regard?
Maybe something like "NFC-A tag sleep state notification" would be better?

Reply:
It should not failed, so I think your proposition sounds better.

Reply:
Changed.